co-noir/co-noir/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+name = "co-noir"
+version = "0.1.0"
```

Comment:
needs a description

Reply:
No Cargo.toml file has a description so far, so let's do this later